tests/translations.py

```diff
@@ -72,9 +73,14 @@ def set_language(client: Client, lang: str, *, force: bool = True):
     with client:
         if not client.features.language.startswith(lang) or force:
             device.change_language(client, language_data)  # type: ignore
+    set_layout(client)
```

Comment:
nit: maybe inline `set_layout()`? (since it's only used once)
```suggestion
    _CURRENT_TRANSLATION.LAYOUT = client.layout_type
```

Reply:
Thanks, I inlined the code on rebase.